src/nimtenbrew/ctr/romfs.nim

```diff
@@ -0,0 +1,33 @@
+import binarylang
+
+## RomFS Header
+## https://www.3dbrew.org/wiki/RomFS#Format
+struct(*romfs, endian = l):
+    s: *magic = "IVFC"
+    u32: *magicNumber = 0x10000
+    u32: masterHashSize
+
+    u64: levelOneLogicOffset
```

Comment:
This might be cleaner if there was a "level" struct with the offset, hashsize, block size, and reserved. Then you'd just have level one, level two, and level 3

Reply:
That's actually kinda smart. I didn't think about that. Will do!